pkg/codegen/schema/schema.go

```diff
@@ -147,6 +147,8 @@ type Enum struct {
 	Comment string
 	// Name for the enum.
 	Name string
+	// DeprecationMessage indicates whether or not the value is deprecated.
```

Comment:
Since this isn't a boolean value, it might be worth clarifying that a non-empty string indicates that the enum is deprecated.

Reply:
This is literally the same comment that exists for every other deprecation message in this file.